contracts/cw-core/src/msg.rs

```diff
@@ -51,6 +51,9 @@ pub struct InitialItem {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct InstantiateMsg {
+    /// Optional Admin with the ability to execute DAO messages directly
+    /// Useful for building SubDAOs completely controlled by a parent DAO.
+    pub admin: Option<Addr>,
```

Comment:
Sorry I didn't catch this earlier - the type here needs to be `Option<String>` and then we need to validate it with `let admin = msg.admin.map(|human| deps.api.addr_validate(&human)).transpose()?;` in the instantiate method.

`Addr` types inside of messages will not be validated so this could currently set the admin to a invalid address.

Reply:
Also need to run `cargo schema` to get this into the schema files.